docs/en/Configuring-Alluxio-with-S3.md

```diff
@@ -114,6 +114,16 @@ dependency to your application with:
 
 {% include Configuring-Alluxio-with-S3/dependency.md %}
 
+## Avoiding Conflicting Client Dependencies
+
+The jets3t and aws-sdk s3 clients all have dependencies on common libraries such as HTTP libraries.
+These dependencies are usually not in conflict with other projects, but in cases like using Apache
+MapReduce with the S3A client, conflicting versions may cause issues at runtime. You can resolve
+this conflict enabling ufs delegation, `alluxio.user.ufs.delegation.enabled=true`, which delegates
+client operations to the under storage through Alluxio servers. Alternatively you can manually
+resolve the conflicts when generating the MapReduce classpath and/or jars, keeping only the highest
```

Comment:
This will probably vary from application to application and isn't straightforward. Given that, woulds you prefer I just remove this comment?